pkg/ccl/changefeedccl/sink_kafka_v2.go

```diff
@@ -340,6 +342,7 @@ func makeKafkaSinkV2(
 	settings *cluster.Settings,
 	mb metricsRecorderBuilder,
 	knobs kafkaSinkV2Knobs,
+	netMetrics *cidr.NetMetrics,
```

Comment:
instead of passing it in separately, pls put this in the metricsRecorder/Builder interface, which presumably is why you put it in sliMetrics in the first place

Reply:
Fixed - I had meant to use this intead.